### ISSUE TITLE
event dragging is wrongfully touch-like on computers with touch support enabled

### ISSUE THREAD

**Issue by skekasaurus** [fullcalendar/fullcalendar#3154]
Hi,

   On the fullcalendar homepage, I can't drag and drop any events on the demo. IE & Firefox work fine for me. Tried on chrome versions "49.0.2623.112 m" and "50.0.2661.94 m" with a Windows 10 PC

**Comment by fierytone** [comment]
I can confirm this is not working in chrome in version 1.3.1

**Comment by skekasaurus** [comment]
I retried the home page on my work PC and it works as expected using chrome (v50.0.2661.94 m). Perhaps this is an issue specific to Windows 10?

**Comment by githubjeka** [comment]
on Win10 Chrome v50.0.2661.94 m Demo Drag&Drop works fine.

**Comment by felixkiss** [comment]
We have one computer running Windows 8, Chrome 50.0.2661.94 m where Demo Drag&Drop doesn't work either.

Is there some kind of feature-detection happening to disable certain features?

**Comment by Amedama96** [comment]
This problem occurs If your machine supports touch events.
To escape this issue, go to chrome's experiments page (chrome://flags). then set Disabled on "Enable touch events" feature.

**Comment by skekasaurus** [comment]
Is there a way to manage this without requiring users to set the flag themselves? My home laptop is indeed a touch screen.

**Comment by K-Y-L-E** [comment]
I found that changing line 28 of fullcalendar.js
from
```
FC.isTouchEnabled = ('ontouchstart' in document) &&
	!window.__karma__; // workaround for phantomjs falsely reporting touch
```
to something like
```
FC.isTouchEnabled = /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent) &&
	!window.__karma__; // workaround for phantomjs falsely reporting touch
```

works with chrome and ipad until someone submits a better, permanent solution

**Comment by gbtux** [comment]
I can confirm too...on a Lenovo Win 10 hybrid PC with touch support.
Thx for the hack. It works perfectly.

Maybe an option to override default ?

**Comment by arshaw** [comment]
Thanks for the key to reproducing @Amedama96

I understand why this is happening. Will get this fixed in next release as soon as I can. There is actually a hidden `isTouch` option in v2.7.1 you can do this...
```
$('#calendar').fullCalendar({
    isTouch: false
});
```

**Comment by SirDre** [comment]
I think I solve it by allowing touch, select, eventclick and draggable events to work on ultra books (touchscreen laptops). 

Try here: http://www.sirdre.com/apps/cifullcalendar/v3

**Comment by arshaw** [comment]
this is fixed in [v2.7.2](https://github.com/fullcalendar/fullcalendar/releases/tag/v2.7.2). could you please verify?
